pre_commit/commands/run.py

```diff
@@ -13,9 +13,9 @@
 from typing import Sequence
 from typing import Set
 from typing import Tuple
-from unicodedata import east_asian_width
 
 from identify.identify import tags_from_path
+from wcwidth import wcswidth
```

Comment:
`wcwidth` introduces a dependency, I'd rather not -- `unicodedata` is fine for now (this is only accidentally working due to `pytest`)